solutions/fully-configurable/variables.tf

```diff
@@ -16,8 +16,7 @@ variable "existing_resource_group_name" {
 
 variable "prefix" {
```

Comment:
variable is missing `nullable=true` 
Please refer - https://terraform-ibm-modules.github.io/documentation/#/da-implementation-guidelines?id=prefix